main.go

```diff
@@ -45,6 +45,10 @@ func (i InfraConfig) UsePro() bool {
 	return len(i.ProConfig.License) > 0
 }
 
+type OperatorConfig struct {
```

Comment:
This would be easier to manage on the top level I think. We don't need a new struct for a single item.

The inlets-pro struct is so that this is strictly separated.

Reply:
I can change this... 

I've put it on it's own struct, because the toplevel struct is named  `infraConfig`. For me, the term infra has to do with infrastructure and provisioners. As this change is about config the operator in general and not a specific provisioner options it made sense to me having it separated.

Reply:
I got why you separated it, but I don't think it needs a struct for the config value.

Reply:
Ok, I'll move the config to the existing infraConfig... no problema..